pkgs/applications/graphics/gimp/plugins/patches/lensfun-Add-missing-header-file.patch

```diff
@@ -0,0 +1,25 @@
+From ca4511c1a4dd8edabe86e4a943861fda07b7e86c Mon Sep 17 00:00:00 2001
+From: Neil Mayhew <neil_mayhew@users.sourceforge.net>
+Date: Fri, 5 Jul 2019 13:37:39 -0600
+Subject: [PATCH] Add missing header file
+Content-Type: text/plain; charset=utf-8
+
+---
+ src/gimplensfun.cpp | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/src/gimplensfun.cpp b/src/gimplensfun.cpp
+index da2a94c..8e9e12e 100644
+--- a/src/gimplensfun.cpp
++++ b/src/gimplensfun.cpp
+@@ -31,6 +31,7 @@
+ #include <libgimp/gimp.h>
+ #include <libgimp/gimpui.h>
+ 
++#include <exiv2/error.hpp>
+ #include <exiv2/image.hpp>
```

Comment:
Upstream discourages from using individual header files since it breaks build, so you should just use
`#include <exiv2/exiv2.hpp>`